### PR TITLE
Mandate certificate_oidc_issuer for artifact verify action

### PR DESCRIPTION
**Pull request by dkourkouzelis**

### CODE REVIEW

File: src/sign.ts [hunk 1]
```diff
@@ -7,6 +7,11 @@ import { getPlatformInfo, setupEndorctl } from "./utils";
 // Sign options
 function get_sign_options(options: any[]): void {
   const ARTIFACT_NAME = core.getInput("artifact_name");
+  const CERTIFICATE_OIDC_ISSUER = core.getInput("certificate_oidc_issuer");
+  const SOURCE_REPOSITORY_REF = core.getInput("source_repository_ref");
```
Comment (hghmn) [inline src/sign.ts:11]: The inputs `certificate_oidc_issuer` and `source_repository_ref` should be added to `./sign/action.yml`

File: src/sign.ts [hunk 2]
```diff
@@ -16,6 +21,16 @@ function get_sign_options(options: any[]): void {
   }
 
   options.push(`--name=${ARTIFACT_NAME}`);
+
+  if (
+    !ENABLE_GITHUB_ACTION_TOKEN &&
+    !(CERTIFICATE_OIDC_ISSUER && SOURCE_REPOSITORY_REF)
+  ) {
```
Comment (hghmn) [inline src/sign.ts]: Should the options be added below when the inputs are present and ENABLE_GITHUB_ACTION_TOKEN is false?

```ts
options.push(`--source-repository-ref=${SOURCE_REPOSITORY_REF}`);
options.push(`--certificate-oidc-issuer=${CERTIFICATE_OIDC_ISSUER}`);
```

File: src/verify.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { getPlatformInfo, setupEndorctl } from "./utils";
 // Verify options
 function get_verify_options(options: any[]): void {
   const ARTIFACT_NAME = core.getInput("artifact_name");
+  const CERTIFICATE_OIDC_ISSUER = core.getInput("certificate_oidc_issuer");
```
Comment (hghmn) [inline src/verify.ts:10]: The input `certificate_oidc_issuer` should be added to `./verify/action.yml`

File: src/sign.ts (hunk 1 above)
Comment (dkourkouzelis) [reply src/sign.ts:11]: Thank you for the thorough review! I think I've addressed it all.

**Review by hghmn — APPROVED** [review]

**Pull request merged**